chapter_post_and_database.asciidoc

```diff
@@ -746,7 +725,6 @@ other methods for your own purposes:
 ----
 ====
 
-// SEBASTIAN: Why not use underscore to emphasize these methods are helper & pretty much private at the same time?
```

Comment:
@Enforcer as "luck" would have it i'm using unittest at work agin these days, and so I found myself considering this exact question, since i'm much more into using underscorey methods to identify private things these days.

in the end i decided it's not really necessary in test classes, because by convention, we already know that the "public" methods are the test methods that start with `test_`, so anything else is already "private".

i mean, unless you start using inheritance in your test classes,  but then you're in deep deep trouble anyways.

so in this case i decided the underscores aren't worth the readability hit...